src/Apps/Artwork/Components/ArtworkSidebar2/ArtworkSidebar2.tsx

```diff
@@ -79,6 +80,12 @@ export const ArtworkSidebar2: React.FC<ArtworkSidebarProps> = ({
 
   const lotLabel = artwork.isBiddable ? artwork.saleArtwork?.lotLabel : null
 
+  const shouldDisplayArtsyGuarantee =
+    !checkIfArtworkIsOnLoanOrPermanentCollection(artwork.saleMessage) &&
+    !isSold &&
+    !isInAuction &&
+    isOfferableFromInquiry
```

Comment:
**question**: Will this be `true` even if `isInquireable=false`?

Reply:
as far as I understood isInquireable is unrelated, but will add some reviewers from NX just to make sure